%YAML:1.0
---
cameraMatrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 7.2584872920265861e+03, 0., 1.5450106113985296e+02, 0.,
       1.5083463754018205e+03, 2.0848863302727489e+02, 0., 0., 1. ]
distCoeffs: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ 7.4209736296358013e+00, -3.9655622474975826e+02,
       9.2172632926134507e-02, 1.0807849973288827e-01, 0. ]
rvecs:
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 1.9522032891273240e+00, 1.2151188584760232e-01,
          2.2996274562299410e+00 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 2.0284012688825852e+00, -1.2742146320281025e-01,
          2.4198231387614992e+00 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 2.0350863080128652e+00, -5.8668532080835126e-02,
          2.4130470908378578e+00 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 2.0636675868807521e+00, -1.4227006184155344e-01,
          2.4336777913485332e+00 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ -2.0047096864033853e+00, 8.8170629310111923e-02,
          -2.3408496614824248e+00 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ -2.0104176077622422e+00, 1.4635020722951525e-01,
          -2.3497268768554291e+00 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ -1.3510373342500903e-01, -1.4319748436125566e+00,
          1.1242081642024043e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 2.0591116158758025e+00, -1.0942306245305382e-01,
          2.4037420934649316e+00 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 2.1531042582806297e+00, -2.7710646242946513e-01,
          2.5317568489901974e+00 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 1.9796912831032927e+00, 2.2271386132938970e-02,
          2.3539227360425254e+00 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 2.0767538817947773e+00, -1.4329224845074129e-01,
          2.4390269327778649e+00 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 2.0773823319386362e+00, -2.0989785771637201e-01,
          2.4467630958615776e+00 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 2.0696916269860717e+00, -1.4329712765467237e-01,
          2.4503890907738861e+00 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 2.0306953646595112e+00, -1.2213585960410013e-01,
          2.4016471374323833e+00 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ -1.8966694126218195e+00, 3.0626690120412475e-01,
          -2.2320098639272286e+00 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ -8.8797083313418967e-01, -1.0550401780030323e+00,
          1.2864186184927651e+00 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 2.0270955565776991e+00, -1.1708212725405209e-01,
          2.4386727563827311e+00 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ -2.0964774283236025e+00, -3.6142252073856011e-02,
          -2.4787552210397297e+00 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 2.0214288546321115e+00, -2.3516065359450781e-02,
          2.3778793848527444e+00 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 1.9879432010154738e+00, -2.2209974317477385e-01,
          -2.2640464813796153e+00 ]
tvecs:
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 7.4205511501677064e+00, 1.5609625091430865e+02,
          1.1270561138123417e+03 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 5.3625496297966961e+00, 4.8271939097981239e+01,
          1.2062292677572607e+03 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 4.0249575151363262e+00, 1.4179932610290174e+02,
          1.1092991609191020e+03 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 2.9328836528387900e+00, 8.3691443514765254e+01,
          1.4187879298069554e+03 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ -5.6268026825112682e-01, 2.1795167528362649e+02,
          1.5184416185485275e+03 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 4.3978969555413894e+00, 5.5310375261585115e+01,
          1.3884727615061486e+03 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ -1.2849085810565882e+01, -6.2461444448930102e+01,
          9.4502042152600370e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 8.1493602719467599e+00, 8.7493260371305936e+01,
          1.7013624348944804e+03 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ -9.5776023168951874e-01, 1.0369629993801105e+02,
          1.3380556315304702e+03 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 3.6288565716294614e+00, 1.4006790972966053e+02,
          1.3035720960455321e+03 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 4.4825137984156829e+00, 9.6802127218951583e+01,
          9.9983013495894807e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 5.6986620919957991e+00, 1.3308267297043999e+01,
          1.3012801667890797e+03 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 4.0436037923069240e-01, 1.1125114826986756e+02,
          1.2325412690654414e+03 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 9.9266060742993361e+00, 2.4932436501994449e+01,
          1.3055735319621451e+03 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 5.7352257050249844e-01, 4.7121905678181811e+01,
          1.2757150142036842e+03 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 5.2869230319201698e+01, -1.8706349151490949e+02,
          2.1301159754964197e+03 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 5.3750932522930723e+00, 1.1447439088515154e+02,
          1.3231709378106989e+03 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 9.7219289597493468e+00, 1.0252572175681898e+02,
          1.3634476434840312e+03 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 6.1608878587022247e+00, 1.1316948970150800e+02,
          1.3850781638343005e+03 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: d
      data: [ 2.3382789773123580e+00, 1.1662481605552881e+02,
          1.1851298573304352e+03 ]
